### PR TITLE
send Pro Bono Applicaiton emails to SUPPORT_EMAIL

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

File: corehq/apps/domain/forms.py [hunk 1]
```diff
@@ -1157,7 +1157,7 @@ def process_submission(self, domain=None):
             }
             html_content = render_to_string("domain/email/pro_bono_application.html", params)
             text_content = render_to_string("domain/email/pro_bono_application.txt", params)
-            recipient = settings.BILLING_EMAIL
+            recipient = settings.SUPPORT_EMAIL
             subject = "[Pro-Bono Application]"
             if domain is not None:
                 subject = "%s %s" % (subject, domain)
```
Comment (dannyroberts) [inline corehq/apps/domain/forms.py]: This is inside `ProBonoForm. process_submission` if that's not clear from the diff

**Comment by dannyroberts** [comment]
@nickpell @amsagoff my attempt at http://manage.dimagi.com/default.asp?167968. Let me know if I got the wrong place

**Comment by dannyroberts** [comment]
@amsagoff signed off with me

**Comment by nickpell** [comment]
:+1:

**Pull request merged**